HW02/Pavel_Karenda/pavel_karenda_hw_02_t_03.rb

```diff
@@ -0,0 +1,50 @@
+require 'time_difference'
+require 'time'
+
+CORE = /Calling core with action/.freeze
+DATE = /^([0-9]*-[0-9]*-[0-9]* [0-9]*:[0-9]*:[0-9]*.[0-9])/.freeze
+arr_lines = []
+
+if ARGV.length != 1
+  puts 'We need exactly one parameter. The name of a file.'
+  exit
+end
+filename = ARGV[0]
+puts "Going to open '#{filename}'"
+
+fh = File.open(filename)
+
+fh.each do |line|
+  arr_lines.push(line) if line.match(CORE)
+end
+fh.close
```

Comment:
Think about the name of a variable; it's a bad name. The variable should be understandable, I don’t understand what it stores.

```suggestion
NEW_NAME= File.open(filename)

NEW_NAME.each do |line|
  arr_lines.push(line) if line.match(CORE)
end
NEW_NAME.close
```